core/src/main/java/hudson/cli/CliManagerImpl.java

```diff
@@ -50,10 +50,11 @@
     
     private Authentication transportAuth;
 
+    //TODO: Migrate the code to Callable decorator
```

Comment:
CalllableFilter is deprecated in remoting. Ideally the code here should be rewritten